src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/ErrorMessages.cs

```diff
@@ -14,5 +14,9 @@ internal class ErrorMessages
         public const string SnapshotReferenceInvalidJsonProperty = "Invalid snapshot reference format for key '{0}' (label: '{1}'). The '{2}' property must be a string value, but found {3}.";
         public const string SnapshotReferencePropertyMissing = "Invalid snapshot reference format for key '{0}' (label: '{1}'). The '{2}' property is required.";
         public const string SnapshotInvalidComposition = "{0} for the selected snapshot with name '{1}' must be 'key', found '{2}'.";
+        public const string ConnectionConflict = "Cannot connect to both Azure App Configuration and Azure Front Door at the same time.";
+        public const string AfdConnectionConflict = "Cannot connect to multiple Azure Front Doors.";
+        public const string AfdLoadBalancingUnsupported = "Load balancing is not supported when connecting to Azure Front Door.";
```

Comment:
Error message can be updated to match https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/223#discussion_r2500578331